src/common/serialization/CMakeLists.txt

```diff
@@ -1,5 +1,22 @@
 MESSAGE(STATUS "Generating protobuf files.")
 
+find_package(Protobuf)
+IF(Protobuf_FOUND)
+    include_directories(${Protobuf_INCLUDE_DIRS})
+ELSE()
+    include(FetchContent)
+    FetchContent_Declare(
+            protobuf
+            GIT_REPOSITORY https://github.com/protocolbuffers/protobuf.git
+            GIT_TAG        "v3.20.1"
+            SOURCE_SUBDIR cmake
+    )
+    set(protobuf_BUILD_TESTS OFF)
+    FetchContent_MakeAvailable(protobuf)
+    include_directories(${CMAKE_SOURCE_DIR}/libs/protobuf/protobuf/include)
```

Comment:
We shouldn't include protobuf libraries.
Serialization using only protobuf compiler for code generation, so generated files shouldn't be committed.